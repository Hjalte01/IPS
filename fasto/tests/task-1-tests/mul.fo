fun int write_int(int x) = write(x)

fun [int] write_int_arr([int] x) = map(write_int, x)

fun [int] main() =
    let x=4*2 in
    let y=1-2*2*2 in
    let z=4*(0-2) in
    let w=(0-4)*(0-4) in
    let v=4*2/4 in
    let q=2/4*4 in
    write_int_arr({x, y, z, w, v, q}) 

    