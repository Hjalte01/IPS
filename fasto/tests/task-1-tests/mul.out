8 -7 -8 16 2 0